docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+<img src="./media/view_Figure_Scale.gif" width="100%">
+</td>
+</tr>
+</table>
 
-```
-TextLabel label = new TextLabel( "Hello World" );
-label.BackgroundColor = Color.Red;
-```
 
-**Figure: TextLabel object with a red background**
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `View`s using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableView`s for each direction:
+
+- [View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+- [View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+- [View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+- [View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When an arrow key is pressed and the current `View` has next `FocusableView` for that direction, the system passes focus to the next `FocusableView`. However, if the current `View` does not have the next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property returns `NULL` and no change occurs.
 
-![TextLabel object with a red background](./media/background_textlabel.png)
 
 <a name="3"></a>
-## Setting the Background Image
+## View Event
 
-You can set a background image of a view:
+`View` provides a variety of `event`s that are commonly used by the components. The following table lists the basic events provided by `View`:
 
+**Table: View events**
+
+| Event               | Description                                               |
+| ------------------- | --------------------------------------------------------- |
+| `KeyEvent`          | Triggered when the key input is received.                   |
+| `TouchEvent`        | Triggered when the touch input is received.                 |
+| `HoverEvent`        | Triggered when the hover input is received.                 |
+| `WheelEvent`        | Triggered when the wheel input is received.                 |
+| `FocusGained`       | Triggered when the control gets the key input focus.        |
+| `FocusLost`         | Triggered when the control loses key input focus.           |
+| `Relayout`          | Triggered after the size has been set on View during relayout. |
+| `LayoutDirectionChanged` | Triggered when the layout direction property of View or its parent View is changed. |
+| `AddedToWindow`     | Triggered after View has been connected to the window.  |
+| `RemovedFromWindow` | Triggered after View has been disconnected from the window. |
+| `VisibilityChanged` | Triggered when the visible property of this or a parent View is changed. |
```

Comment:
Here usage of "this" can confuse the reader.
Request you to consider the previous comment, https://github.com/Samsung/tizen-docs/pull/962#discussion_r353617662.

Reply:
Thanks, I changed it as "View or parent View".